### ISSUE TITLE
Embedding <script> tags via a Radius snippet can blow up the Mercury editor due to improper character escaping.

### ISSUE THREAD

**Issue by nreckart** [wvuweb/cleanslate-cms#119]
Found this out while trying out different methods of adding a Wufoo form to a page manually using the code snippet they provide. Pasting the code into a Radius snippet and then saving/publishing the page works. You can view the Wufoo form on the public version of the page, but if you reload the editor window, it blows up with the following JS error:

`SyntaxError: unterminated string literal`

Commit wvuweb/cleanslate@1eeb33b8bf9f70d9 dealt with a similar issue in the region history modal.

**Comment by zeroedin** [comment]
wvuweb/cleanslate@1eeb33b (typo)

**Comment by dmolsen** [comment]
tip: you can add `?w=0` to the end of a commit URL to remove whitespace-only changes.

**Comment by nreckart** [comment]
The issue is with the way Mercury is currently setup to load/initialize the data for existing snippets on a page so they can be edited. I've tried several ways to fix it without success. I then tried some alternate methods of getting a `<script>` tag into an editable region and discovered that doing so has some undesired affects. Basically, any `<script>` tags within an editable region get executed repeatedly anytime the content is edited. So for every key press, all scripts in the region get reloaded. I haven't been able to track down why, but it's either Mercury doing it or perhaps it's just the way contenteditable works. Either way, it's out of scope for now. 

Long story short, `<script>` tags should not be used in editable regions, either via snippets or manual editing of HTML. So, with respect to this particular issue, I will be escaping any `<script>` tags that may appear in the Radius snippet content, which will fix the error, but will also render the scripts useless.

If there turns out to be a valid use case for embedding scripts within a page that can't already be done with the page javascript feature, we'll need to look into that separately.
